theseus/src/state/profiles.rs

```diff
@@ -23,6 +25,8 @@ pub const CURRENT_FORMAT_VERSION: u32 = 1;
 // Represent a Minecraft instance.
 #[derive(Serialize, Deserialize, Clone, Debug)]
 pub struct Profile {
+    #[serde(default = "create_uuid")]
+    pub uuid: Uuid, // todo: will be used in restructure to refer to profiles
```

Comment:
this shouldn't have a default (see below)- for existing profiles we can probs just nuke since no one uses the launcher yet